### DISCUSSION TITLE
ww3_multi with rectilinear and unstructured grids

### DISCUSSION THREAD

**Discussion by SubgridFeatures** [NOAA-EMC/WW3#756] [Q&A]
Hello everyone. Can i run the model with ww3_multi if I have two grids: global rectilinear and local unstructured or in that case are used consequence ww3_shel? First run ww3_shel for global, then are created nest.ww3 for unstructured grid, and then run ww3_shel for unst grid. Also there is wind from nc file in rectilinear coordinate. Are these data reqiured only for global grid or it goes to unstructured grid from global? Thank a lot

**Comment by aliabdolali** [comment]
@SubgridFeatures THe multi does not support the combination of rectilinear and unstr. You should follow the second path you described.
Also, ww3_prnc reads the nc file in rectilinear coordinate and takes care of interpolating it on either global grid or unstr mesh.

**Comment by mickaelaccensi** [comment]
From what I remember, you can run ww3_multi with different kind of grids (rect, cuv, unst) if you set a different rank ID for each of them

**Reply by JessicaMeixner-NOAA** [reply]
Even for unstructured? I didn't know you could run more than one grid with unstructured in multi-grid at this time.
